aws/table_aws_guardduty_detector.go

```diff
@@ -91,7 +98,7 @@ func tableAwsGuardDutyDetector(_ context.Context) *plugin.Table {
 				Description: resourceInterfaceDescription("akas"),
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getAwsGuardDutyDetectorAkas,
-				Transform:   transform.FromValue(),
+				Transform:   transform.FromValue().Transform(arnToAkas),
```

Comment:
Use `transform.EnsureStringArray` transformation to form the `akas`.
```suggestion
				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
```